### DISCUSSION TITLE
A black box formula is still in use in entity“P_cell_int_factor”

### DISCUSSION THREAD

**Discussion by zjupharma** [Open-Systems-Pharmacology/Forum#1638] [Questions & Problems]
Hello! I add a subcompartment using intracellular as the templet within the lung tissue and build the neighborhood between interstitial and it in MoBi 11. Then, i load the parameter from the neighborhood lung_int_cell. When these are finished and I tried to update from building block in simulation, several similar errors appear: A black box formula is still in use in entity“P_cell_int_factor”. Dose someone know how to solve this problem, please let me know.
Best regards,
Yu Wang
![error](https://github.com/Open-Systems-Pharmacology/Forum/assets/109033288/abd4c048-ce40-49c6-bf66-c9fd0eb1fc39)

**Comment by Xiaoyang-svg** [comment]
Hello, did you solve this problem?

**Comment by rwmcintosh** [comment]
In your added compartment, you should change the formula because it won't be replaced during simulation construction.

![image](https://github.com/user-attachments/assets/c2ed5580-531f-469d-a821-2de229aeecdc)
